src/utils/feature-flag-utils.ts

```diff
@@ -9,15 +8,12 @@ declare const window: any
  * Uses "business-create" project (per LD client id in config).
  */
 const defaultFlagSet: LDFlagSet = {
-  'supported-filings': [
-    FilingTypes.DISSOLUTION,
-    FilingTypes.INCORPORATION_APPLICATION,
-    FilingTypes.REGISTRATION
-  ],
```

Comment:
This default would have allowed these filing types in Prod if the user couldn't talk to LaunchDarkly to fetch the latest FFs. This is incorrect -- if the user can't talk to LD then the default FFs should disable the new features.